### PR TITLE
Synonyms getting dropped when making component

### LINKED ISSUE #656: Synonyms getting dropped when making component
## Overview
Example case: DOID:635, `AIDS` exactSynonym getting dropped.

<details><summary>Component download</summary>
<p>

```owl
    <owl:Class rdf:about="http://purl.obolibrary.org/obo/DOID_635">
        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/DOID_526"/>
        <rdfs:subClassOf>
            <owl:Restriction>
                <owl:onProperty rdf:resource="http://purl.obolibrary.org/obo/RO_0002451"/>
                <owl:someValuesFrom>
                    <owl:Class>
                        <owl:unionOf rdf:parseType="Collection">
                            <rdf:Description rdf:about="http://purl.obolibrary.org/obo/TRANS_0000006"/>
                            <rdf:Description rdf:about="http://purl.obolibrary.org/obo/TRANS_0000007"/>
                            <rdf:Description rdf:about="http://purl.obolibrary.org/obo/TRANS_0000011"/>
                            <rdf:Description rdf:about="http://purl.obolibrary.org/obo/TRANS_0000012"/>
                        </owl:unionOf>
                    </owl:Class>
                </owl:someValuesFrom>
            </owl:Restriction>
        </rdfs:subClassOf>
        <obo:IAO_0000115 xml:lang="en">A Human immunodeficiency virus infectious disease that results_in reduction in the numbers of CD4-bearing helper T cells below 200 per microliter of blood or 14% of all lymphocytes thereby rendering the subject highly vulnerable to life-threatening infections and cancers, has_material_basis_in Human immunodeficiency virus 1 or has_material_basis_in Human immunodeficiency virus 2, which are transmitted_by sexual contact, transmitted_by transfer of blood, semen, vaginal fluid, pre-ejaculate, or breast milk, transmitted_by congenital method, and transmitted_by contaminated needles. Opportunistic infections are common in people with AIDS.</obo:IAO_0000115>
        <oboInOwl:hasDbXref>EFO:0000765</oboInOwl:hasDbXref>
        <oboInOwl:hasDbXref>ICD10CM:B20</oboInOwl:hasDbXref>
        <oboInOwl:hasDbXref>MESH:D000163</oboInOwl:hasDbXref>
        <oboInOwl:hasDbXref>NCI:C2851</oboInOwl:hasDbXref>
        <oboInOwl:hasDbXref>SNOMEDCT_US_2023_03_01:62479008</oboInOwl:hasDbXref>
        <oboInOwl:hasDbXref>UMLS_CUI:C0001175</oboInOwl:hasDbXref>
        <oboInOwl:hasExactSynonym xml:lang="en">AIDS</oboInOwl:hasExactSynonym>
        <oboInOwl:hasExactSynonym xml:lang="en">acquired Immune deficiency</oboInOwl:hasExactSynonym>
        <oboInOwl:hasOBONamespace>disease_ontology</oboInOwl:hasOBONamespace>
        <oboInOwl:hasRelatedSynonym xml:lang="en">acquired immune deficiency syndrome</oboInOwl:hasRelatedSynonym>
        <oboInOwl:id>DOID:635</oboInOwl:id>
        <oboInOwl:inSubset rdf:resource="http://purl.obolibrary.org/obo/doid#DO_infectious_disease_slim"/>
        <oboInOwl:inSubset rdf:resource="http://purl.obolibrary.org/obo/doid#NCIthesaurus"/>
        <oboInOwl:inSubset rdf:resource="http://purl.obolibrary.org/obo/doid#zoonotic_infectious_disease"/>
        <rdfs:label xml:lang="en">acquired immunodeficiency syndrome</rdfs:label>
        <skos:exactMatch>MESH:D000163</skos:exactMatch>
    </owl:Class>
    <owl:Axiom>
        <owl:annotatedSource rdf:resource="http://purl.obolibrary.org/obo/DOID_635"/>
        <owl:annotatedProperty rdf:resource="http://purl.obolibrary.org/obo/IAO_0000115"/>
        <owl:annotatedTarget xml:lang="en">A Human immunodeficiency virus infectious disease that results_in reduction in the numbers of CD4-bearing helper T cells below 200 per microliter of blood or 14% of all lymphocytes thereby rendering the subject highly vulnerable to life-threatening infections and cancers, has_material_basis_in Human immunodeficiency virus 1 or has_material_basis_in Human immunodeficiency virus 2, which are transmitted_by sexual contact, transmitted_by transfer of blood, semen, vaginal fluid, pre-ejaculate, or breast milk, transmitted_by congenital method, and transmitted_by contaminated needles. Opportunistic infections are common in people with AIDS.</owl:annotatedTarget>
        <dc:type rdf:resource="http://purl.obolibrary.org/obo/ECO_0007638"/>
        <dc:type rdf:resource="http://purl.obolibrary.org/obo/ECO_0007643"/>
        <oboInOwl:hasDbXref>url:http://en.wikipedia.org/wiki/AIDS</oboInOwl:hasDbXref>
        <oboInOwl:hasDbXref>url:http://www2.merriam-webster.com/cgi-bin/mwmednlm?book=Medical&amp;va=aids</oboInOwl:hasDbXref>
    </owl:Axiom>
    <owl:Axiom>
        <owl:annotatedSource rdf:resource="http://purl.obolibrary.org/obo/DOID_635"/>
        <owl:annotatedProperty rdf:resource="http://www.geneontology.org/formats/oboInOwl#hasExactSynonym"/>
        <owl:annotatedTarget xml:lang="en">AIDS</owl:annotatedTarget>
        <oboInOwl:hasSynonymType rdf:resource="http://purl.obolibrary.org/obo/OMO_0003012"/>
    </owl:Axiom>
```

</p>
</details> 

<details><summary>Component</summary>
<p>

```owl
    <owl:Class rdf:about="http://purl.obolibrary.org/obo/DOID_635">
        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/DOID_526"/>
        <oboInOwl:hasDbXref>EFO:0000765</oboInOwl:hasDbXref>
        <oboInOwl:hasDbXref>ICD10CM:B20</oboInOwl:hasDbXref>
        <oboInOwl:hasDbXref>MESH:D000163</oboInOwl:hasDbXref>
        <oboInOwl:hasDbXref>NCI:C2851</oboInOwl:hasDbXref>
        <oboInOwl:hasDbXref>UMLS_CUI:C0001175</oboInOwl:hasDbXref>
        <oboInOwl:hasExactSynonym xml:lang="en">acquired Immune deficiency</oboInOwl:hasExactSynonym>
        <oboInOwl:hasExactSynonym xml:lang="en">acquired immunodeficiency syndrome</oboInOwl:hasExactSynonym>
        <oboInOwl:hasRelatedSynonym xml:lang="en">acquired immune deficiency syndrome</oboInOwl:hasRelatedSynonym>
        <rdfs:label xml:lang="en">acquired immunodeficiency syndrome</rdfs:label>
        <skos:exactMatch>MESH:D000163</skos:exactMatch>
    </owl:Class>
```

</p>
</details>

### PR DESCRIPTION
**Pull request by joeflack4**
> [!warning]
> This PR entails:
> - #663 
> 
> Needed to merge that, otherwise build was failing. 

Resolves #656

## Overview
Bug fix: Synonym drops. Synonyms with type OMO:0003012 were getting dropped because there was not an entry for this in properties.txt.

### Documentation
Was the documentation added/updated under `docs/`?
- [ ] Yes
- [x] No, updates to the docs were not necessary after careful consideration

### QC
Was the full pipeline run before submitting this PR using `sh run.sh make build-mondo-ingest` on this branch (after 
`docker pull obolibrary/odkfull:dev`), and no errors occurred?
- [x] Yes
- [ ] No, there are no functional (code-related) changes to the pipeline in the PR, so no re-run is necessary
   
Build PR:
- #666

### New Packages
Were any new *Python packages* added?
- [ ] Yes, [requirements.txt.full](https://github.com/INCATools/ontology-development-kit/blob/master/requirements.txt.full) was updated and an [ODK issue](https://github.com/INCATools/ontology-development-kit/issues/new) submitted
- [x] No

Were any other *non-Python packages* added?
- [ ] Yes, [Dockerfile](https://github.com/INCATools/ontology-development-kit/blob/master/Dockerfile) updated and an [ODK issue](https://github.com/INCATools/ontology-development-kit/issues/new) submitted
- [x] No

### PR Review and Conversations Resolved
Has the PR been sufficiently reviewed by at least 1 team member of the Mondo Technical team and all threads resolved?
- [x] Yes

### CODE REVIEW

File: src/ontology/config/property-map.sssom.tsv
Comment (twhetzel) [inline src/ontology/config/property-map.sssom.tsv:10]: Remember to update this branch with updates from https://github.com/monarch-initiative/mondo-ingest/pull/663 so this line will be removed.

Comment (joeflack4) [reply src/ontology/config/property-map.sssom.tsv]: Done!

**Review by twhetzel — APPROVED** [review]
Changes and build look good

File: src/ontology/mondo-ingest.Makefile
Comment (joeflack4) [inline src/ontology/mondo-ingest.Makefile:76]: [Trish observed](https://github.com/monarch-initiative/mondo-ingest/pull/666#discussion_r1769647426) unintended consequences of a harmful `robot remove` line that I'd tested out and added here and forgot to remove. I've since removed and am rerunning builds.

Comment (joeflack4) [reply src/ontology/mondo-ingest.Makefile:76]: RE Trish:
> Changes and build look good

Sounds good. Gonna hold off on merging until #663 is approved, as this PR of necessity includes those changes.

**Review by twhetzel — APPROVED** [review]
Approving since related PR looks good, https://github.com/monarch-initiative/mondo-ingest/pull/664.

**Pull request merged**
